_CoqProject

```diff
@@ -0,0 +1,9 @@
+-R UniMath
```

Comment:
I believe it should be `-Q` instead of `-R`. @rmatthes : do you know more?

Reply:
Once this comment is resolved, we can merge this PR.